pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -429,5 +401,65 @@ function createTooltip(el) {
         .style("padding", "10px")
         .style("line-height", "1.3")
         .style("z-index", 1)
-        .style("font", "11px sans-serif")
+        .style("pointer-events", "none")
+        .style("font", "12px sans-serif")
 }
+
+function positionTooltip(x, y) {
+    let tooltipX = x;
+    let tooltipY = y;
+
+    if (x > displayMaxX / 2) {
+        tooltipX = tooltipX - tooltip.property("offsetWidth");
+    }
+
+    if (y > displayMaxY / 2) {
+        tooltipY = tooltipY - tooltip.property("offsetHeight");
+    } else {
+        // It looks really goofy if you don't. 20px is about the size of the mouse on a 1080 scaled display
+        tooltipY += 20
+    }
+
+    tooltip
+        .style("left", tooltipX + "px")
+        .style("top", tooltipY + "px")
+}
+
+function showDetailedTooltip(d, event, parent) {
+    let tooltipX = event.pageX;
+    let tooltipY = event.pageY;
+    if (detailedToolTipIsVisible) {
+        let resourceBarHtml = getResourceBarContent(
+            {
+                title: d.text,
+                kind: d.kind,
+                namespace: d.namespace,
+                time: theTime
+            }
+        );
+        tooltip.html(resourceBarHtml);
+        positionTooltip(tooltipX, tooltipY);
```

Comment:
is this going to cause a flicker (visible for a short time in original location/size, then move/resize based on additional data)?